File: 656_div_3/a.go
package main
import (
  "fmt"
)

func main() {
  var t int
  fmt.Scan(&t)

  for t>0 {
    t--
    var x,y,z,m int

    fmt.Scan(&x,&y,&z)
    m = max(x,y,z)

    if ((m==y && m==z) ||
        (m==x && m==z) ||
        (m==x && m==y)) {
      fmt.Print("YES\n")
      fmt.Printf("%d %d %d\n", m, min(x,y,z), min(x,y,z))
    } else {
      fmt.Print("NO\n")
    }
  }
}

File: 656_div_3/a.py
t = int(input())

for _ in range(t):
    x,y,z = [int(i) for i in input().split()]
    m = max(x,y,z)

    if ((m == x and m == y) or
        (m == y and m == z) or
        (m == x and m == z)):
        print("YES")
        print(m, min(x,y,z), min(x,y,z))
    else:
        print("NO")
